### PR TITLE
chore(refactor): Remove arrow dependency from the project

### LINKED ISSUE #6927: Remove arrow from project
Only 1 line uses the dependency and can be replaced by other dependencies we are using

### PR DESCRIPTION
**Pull request by mansiag**
Fixes #6927 

#### Short description of what this resolves:
Removes arrow dependency from the project by using pytz and humanize libraries for the same functionality

#### Checklist

- [X]   I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The unit tests pass locally with my changes 
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

- [X] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity increasing per file
==============================
- tests/all/integration/api/helpers/test_humanize_helper.py  2
- app/api/helpers/humanize_helper.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904&bid=18234295)

File: tests/all/integration/api/helpers/test_humanize_helper.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from app.api.helpers.humanize_helper import humanize_helper
+from tests.factories.order import OrderFactory
+
+
+class TestHumanizeHelper(OpenEventTestCase): 
+    def test_humanize_helper(self):
+        """Method to test humanization of order creation time"""
+
+        with self.app.test_request_context():
+            test_order = OrderFactory(created_at=datetime.datetime.now()
```
Comment (niranjan94) [inline tests/all/integration/api/helpers/test_humanize_helper.py:12]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904)

File: app/api/helpers/humanize_helper.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+
+import humanize
+import pytz
+
+
+def humanize_helper(time):
+    """Returns time passed from now in a human readable duration"""
+
+    return humanize.naturaldelta(datetime.now(pytz.utc)
```
Comment (niranjan94) [inline app/api/helpers/humanize_helper.py:10]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904)

File: app/api/helpers/humanize_helper.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+import humanize
+import pytz
+
+
+def humanize_helper(time):
+    """Returns time passed from now in a human readable duration"""
+
+    return humanize.naturaldelta(datetime.now(pytz.utc) 
+                - time.astimezone(pytz.utc)) + ' ago'
```
Comment (niranjan94) [inline app/api/helpers/humanize_helper.py:11]: Codacy Issue found: [Final newline missing](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904)

File: tests/all/integration/api/helpers/test_humanize_helper.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+class TestHumanizeHelper(OpenEventTestCase): 
+    def test_humanize_helper(self):
+        """Method to test humanization of order creation time"""
+
+        with self.app.test_request_context():
+            test_order = OrderFactory(created_at=datetime.datetime.now() 
+                                        - datetime.timedelta(days=10))
+            actual_response = humanize_helper(test_order.created_at)
+            expected_response = '10 days ago'
+            self.assertEqual(actual_response, expected_response)
```
Comment (niranjan94) [inline tests/all/integration/api/helpers/test_humanize_helper.py:16]: Codacy Issue found: [Final newline missing](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904)

File: tests/all/integration/api/helpers/test_humanize_helper.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import datetime
+
+from app.api.helpers.humanize_helper import humanize_helper
+from tests.factories.order import OrderFactory
+
+
+class TestHumanizeHelper(OpenEventTestCase):
```
Comment (niranjan94) [inline tests/all/integration/api/helpers/test_humanize_helper.py:7]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904)

Comment (iamareebjamal) [inline tests/all/integration/api/helpers/test_humanize_helper.py:7]: Please use new pytest style test

File: app/api/helpers/humanize_helper.py (hunk 3 above)
Comment (iamareebjamal) [inline app/api/helpers/humanize_helper.py:11]: Humanize adds ago automatically

**Comment by iamareebjamal** [comment]
Remove arrow from requirements as well

File: app/api/helpers/humanize_helper.py (hunk 3 above)
Comment (mansiag) [reply app/api/helpers/humanize_helper.py:11]: it adds ago with naturaltime, not with naturaldelta .

Comment (iamareebjamal) [reply app/api/helpers/humanize_helper.py:11]: You can use naturaltime with timedelta

`humanize.naturaltime(dt.timedelta(seconds=1001)) == 16 minutes ago`

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/all/integration/api/helpers/test_humanize_helper.py  2
- app/api/helpers/humanize_helper.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904&bid=18234295)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/all/integration/api/helpers/test_humanize_helper.py  2
- app/api/helpers/humanize_helper.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904&bid=18234295)

File: app/templates/flask_ext/jinja/filters.py [hunk 6]
```diff
@@ -21,4 +21,5 @@ def simple_datetime_display(date):
     def humanize_filter(time):
         if not time:
             return "N/A"
-        return humanize_helper(time)
\ No newline at end of file
+        return humanize_helper(time)
+
```
Comment (iamareebjamal) [inline app/templates/flask_ext/jinja/filters.py]:
```suggestion

```

File: app/api/helpers/humanize_helper.py [hunk 7]
```diff
@@ -0,0 +1,10 @@
+from datetime import datetime
+
+import humanize
+import pytz
+
+
+def humanize_helper(time):
```
Comment (iamareebjamal) [inline app/api/helpers/humanize_helper.py:7]: No need to create a new file for it, add this function in `filters.py`

File: app/templates/flask_ext/jinja/filters.py [hunk 8]
```diff
@@ -20,5 +21,5 @@ def simple_datetime_display(date):
     def humanize_filter(time):
         if not time:
             return "N/A"
```
Comment (iamareebjamal) [inline app/templates/flask_ext/jinja/filters.py:23]: You can move this logic to the function as well and add a test for it too

File: tests/all/integration/api/helpers/test_humanize_helper.py [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+def test_humanize_helper(db):
+    """Method to test humanization of order creation time"""
+
+    test_order = OrderFactory(
+        created_at=datetime.datetime.now() - datetime.timedelta(days=10)
+    )
+    actual_response = humanize_helper(test_order.created_at)
+    expected_response = '10 days ago'
+    assert actual_response == expected_response
+
```
Comment (niranjan94) [inline tests/all/integration/api/helpers/test_humanize_helper.py:16]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- tests/all/integration/api/helpers/test_humanize_helper.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904&bid=18234295)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- tests/all/integration/api/helpers/test_humanize_helper.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904&bid=18234295)

File: tests/all/integration/api/helpers/test_humanize_helper.py [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
```
Comment (niranjan94) [inline tests/all/integration/api/helpers/test_humanize_helper.py:16]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/all/integration/api/helpers/test_humanize_helper.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5559904&bid=18234295)

**Pull request merged**